UnityProject/Assets/Scripts/PlayGroups/Input/InputController.cs

```diff
@@ -97,6 +98,14 @@ private void CheckAltClick()
 			}
 		}
 
+		private void CheckSuicide()
+		{
+			if (UnityEngine.Input.GetKeyDown(KeyCode.Backspace))
+			{
+				PlayerManager.LocalPlayerScript.playerNetworkActions.CmdCommitSuicide();
+			}
+		}

```

Comment:
Wouldn't this kill the player if they use backspace in chat? I don't think this should be a feature here anyway. if you want to commit suicide make it a chat command i.e "/suicide or .suicide" something like that

Reply:
Agreed...

Reply:
@BeastKiller2013 Didn't you plan to remove this one too?
Leave only the PNA in if you like... You could look into adding a chat or editorcommand (like gib-all) for it later ;)

Reply:
I can do that.